remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+import importlib
+import remotespark.utils.configuration as conf
+import remotespark.utils.constants as constants
+
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        module, class_name = conf.events_handler_class().rsplit('.', 1)
```

Comment:
Anyone should use his own handler should add the whole path till the class used as the handler. Else, this will not work (Compilation Error) as we are using importlib_module method which expects module.